add option to write eigenvectors and extra info in bands, useful for … (#154)
stfc/janus-core: README.md

janus phonons --struct tests/data/NaCl.cif --supercell 2x2x2 --minimize --arch mace_mp --calc-kwargs "{'model' : 'small'}"
```

> This will save the Phonopy parameters, including displacements and force constants, to `NaCl-phonopy.yml` and `NaCl-force\_constants.hdf5`, in addition to generating a log file, `phonons.log`, and summary of inputs, `phonons_summary.yml`.
> Additionaly, to calculate band structure and generate `NaCl-auto-bands.yml` use:

Reviewer: ```suggestion
This will save the Phonopy parameters, including displacements and force constants, to `NaCl-phonopy.yml` and `NaCl-force_constants.hdf5`, in addition to generating a log file, `phonons.log`, and summary of inputs, `phonons_summary.yml`.

Additionally, the `--band` option can be added to calculate the band structure and save the results to `NaCl-auto_bands.yml`:
```

```shell

> janus phonons --struct tests/data/NaCl.cif --supercell 2x2x2 --minimize --arch mace_mp --calc-kwargs "{'model' : 'small'}"

Reviewer: ```suggestion
janus phonons --struct tests/data/NaCl.cif --supercell 2x2x2 --minimize --arch mace_mp --calc-kwargs "{'model' : 'small'}" --band
```

```

> If you need eigenvectors and group velocities written add `--write-full` option. Will generate a much bigger bands file but can be
> used to visualise phonon modes.

Reviewer: ```suggestion

If you need eigenvectors and group velocities written, add the `--write-full` option. This will generate a much larger file, but can be used to visualise phonon modes.
```
